modules/nw-ingress-integrating-route-secret-certificate.adoc

```diff
@@ -0,0 +1,71 @@
+//
+// * ingress/routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-integrating-route-secret-certificate_{context}"]
+= Secure route with external certificates in TLS secrets
+
+:FeatureName: Secure route with external certificates in TLS secrets
+include::snippets/technology-preview.adoc[]
+
+The following workflow describes how to integrate third-party certificate management solutions, using the certificate reference field outlined under API Extensions.
+
+A new `.spec.tls.externalCertificate` field has been added to the route's `.spec.tls` configuration. This field can be used to specify the name of a secret containing the certificate data.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the cluster-admin role.
```

Comment:
@lihongan, @lunarwhite I think any user who has the `create` and `update` permission on `custom-host` can use this feature. Is there any explicit requirement of having cluster-admin role?

Reply:
> any user who has the create and update permission on custom-host can use this feature

Yes, it should be good to remove this line of requirement

Reply:
Oh, I didn't notice you mentioned "custom-host". Please omit my previous comment

And yes, it could document something like:
```
You must have "custom-host" create and update permission.
```

Feel free to reword to make it more precise^

Reply:
"Make sure you have the `create` and `update` permissions on the `routes/custom-host`  sub-resource."


Something like this may be. @subhtk please feel free to reword it

Reply:
I cannot use sub-resource. Will this be fine- "Make sure you have the `create` and `update` permissions on the `routes/custom-host`.